tools/convert/README.md

```diff
@@ -0,0 +1,21 @@
+# brick-convert
+
+## Install dependencies
+```bash
+pip install -r requirements.txt
+```
+
+## Help
+```bash
+python convert.py --help     
+```
+
+## Usage
+```bash
+python convert.py --source 1.0.2 --target 1.1 examples/ebu3b_brick.ttl 
```

Comment:
@shreyasnagare  Just want to make sure: is it usable only inside `tools/convert/`?

Reply:
To use it from the root directory, you must use `./tools/convert/convert.py` and `./tools/convert/examples/ebu3b_brick.ttl` instead of `convert.py` and `examples/ebu3b_brick.ttl`. I usually have _the README directory_ as the working directory. We can change the README to have the `./tools/convert/` path prefix in front of all the filenames if you think this is wrong.

Reply:
I just think we can make it executable later. Nothing urgent.